core/templates/services/state-interaction-stats.service.ts

```diff
@@ -70,6 +70,8 @@ const STATE_INTERACTION_STATS_URL_TEMPLATE: string = (
 
 @Injectable({providedIn: 'root'})
 export class StateInteractionStatsService {
+  cachedStats: Promise<IStateRulesStats> = null;
```

Comment:
Shouldn't we be storing the actual stats, rather than the promise?

Reply:
Done.